Golint is complaining about the VERSION var:
`/home/travis/gopath/src/k8s.io/contrib/exec-healthz/pkg/version/version.go:19:5: exported var VERSION should have comment or be unexported`

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,4 +1,7 @@
-sudo: false
+sudo: required
+
+services:
+  - docker
 
 language: go
 

--- a/exec-healthz/Dockerfile.in
+++ b/exec-healthz/Dockerfile.in
@@ -18,4 +18,5 @@ MAINTAINER Prashanth B <beeps@google.com>
 
 ADD bin/ARG_ARCH/ARG_BIN /ARG_BIN
 
+USER nobody:nobody
 ENTRYPOINT ["/ARG_BIN"]

--- a/exec-healthz/Makefile
+++ b/exec-healthz/Makefile
@@ -28,7 +28,7 @@ ARCH ?= amd64
 ### These variables should not need tweaking.
 ###
 
-SRC_DIRS := cmd # directories which hold app source (not vendored)
+SRC_DIRS := cmd pkg # directories which hold app source (not vendored)
 
 ALL_ARCH := amd64 arm arm64 ppc64le
 

--- a/exec-healthz/OWNERS
+++ b/exec-healthz/OWNERS
@@ -2,4 +2,4 @@ assignees:
 - bprashanth
 - luxas
 - mikedanese
-
+- mrhohn

--- a/exec-healthz/cmd/exechealthz/exechealthz.go
+++ b/exec-healthz/cmd/exechealthz/exechealthz.go
@@ -29,6 +29,7 @@ import (
 	"sync"
 	"time"
 
+	"k8s.io/contrib/exec-healthz/pkg/version"
 	"k8s.io/kubernetes/pkg/util/clock"
 	utilexec "k8s.io/kubernetes/pkg/util/exec"
 )
@@ -47,6 +48,7 @@ var (
 	period     = flag.Duration("period", 2*time.Second, "Period to run the given cmd in an async worker.")
 	maxLatency = flag.Duration("latency", 30*time.Second, "If the async worker hasn't updated the probe command output in this long, return a 503.")
 	quiet      = flag.Bool("quiet", false, "Run in quiet mode by only logging errors.")
+	printVer   = flag.Bool("version", false, "Print the version and exit.")
 	// probers are the async workers running the cmds, the output of which is used to service /healthz or others customized pathes.
 	probers = make(map[string]*execWorker)
 )
@@ -166,6 +168,12 @@ func newExecWorker(probeCmd, probePath string, execPeriod time.Duration, exec ut
 
 func main() {
 	flag.Parse()
+
+	if *printVer {
+		fmt.Printf("%s\n", version.VERSION)
+		os.Exit(0)
+	}
+
 	flagCheckUrlCmd()
 	links := []struct {
 		link, desc string

--- a/exec-healthz/pkg/version/version.go
+++ b/exec-healthz/pkg/version/version.go
@@ -1,0 +1,21 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+// VERSION is the app-global version string, which should be substituted with a
+// real value during build.
+var VERSION = "UNKNOWN"

